docs/user-guide/horizontal-pod-autoscaling/walkthrough.md

```diff
@@ -120,6 +127,145 @@ Here CPU utilization dropped to 0, and so HPA autoscaled the number of replicas
 
 **Note** autoscaling the replicas may take a few minutes.
 
+## Autoscaling on multiple metrics and custom metrics
+
+You can introduce additional metrics to use when autoscaling the `php-apache` Deployment
```

Comment:
nope, according to the style guide terms should be CamelCase: https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/contribute/style-guide.md#use-camel-case-for-api-objects